install.sh

```diff
@@ -130,22 +130,22 @@ mkdir -p $INSTALL_TMP
 # local testing of this file.
 if [[ -e releases/$DOWNLOAD ]]; then
   echo "Using existing release: releases/$DOWNLOAD_FILENAME"
-  cp releases/$DOWNLOAD_FILENAME $INSTALL_TMP
+  cp releases/"$DOWNLOAD_FILENAME" $INSTALL_TMP
 else
   buildkite-download "$DOWNLOAD_URL" "$INSTALL_TMP/$DOWNLOAD_FILENAME"
 fi
 
 # Extract the download to a tmp folder inside the $DESTINATION
 # folder
-tar -C $INSTALL_TMP -zxf $INSTALL_TMP/$DOWNLOAD_FILENAME
+tar -C "$INSTALL_TMP" -zxf "$INSTALL_TMP"/"$DOWNLOAD_FILENAME"
 
 # Move the buildkite binary into a bin folder
-mkdir -p $DESTINATION/bin
-mv $INSTALL_TMP/buildkite-agent $DESTINATION/bin
-chmod +x $DESTINATION/bin/buildkite-agent
+mkdir -p "$DESTINATION"/bin
+mv $INSTALL_TMP/buildkite-agent "$DESTINATION"/bin
+chmod +x "$DESTINATION"/bin/buildkite-agent
 
 # Copy the latest config file as dist
-mv $INSTALL_TMP/buildkite-agent.cfg $DESTINATION/buildkite-agent.dist.cfg
+mv "$INSTALL_TMP"/buildkite-agent.cfg "$DESTINATION"/buildkite-agent.dist.cfg
```

Comment:
I was thinking for this case:

```bash
❯ export WITH_SPACE="this is a folder / "                                                                                                    
❯ echo "$WITH_SPACE"/something                                                                                                               
this is a folder / /something
```

(only because in this case it's a path)